docs: encrypt CLI usage (#21162)
renovatebot/renovate: docs/usage/configuration-options.md

## encrypted

> Use this to encrypt secrets in a way which can be stored in repository configs. It is possible to encrypt secrets using either a HTML page, or CLI.

Reviewer: ```suggestion
Before you put any secrets in your repository configuration, encrypt the secrets.
You can encrypt secrets using either a HTML page, or the CLI.
```

> To encrypt secrets for the hosted Mend Renovate app for github.com using a HTML page, go to [app.renovatebot.com/encrypt](https://app.renovatebot.com/encrypt) and complete the form. Alternatively, this form may be downloaded and edited for self-hosted users to use their own PGP public key.

Reviewer: ```suggestion
To encrypt secrets for the hosted Mend Renovate app for github.com with a HTML page, go to [app.renovatebot.com/encrypt](https://app.renovatebot.com/encrypt) and complete the form.
If you're self-hosting Renovate, you may download and edit the form, to use your own PGP public key.
```

> You can also encrypt secrets using the CLI, using the `gpg` command line tool plus some help from `curl` and `jq`. Here is an example:

Reviewer: ```suggestion
You can also encrypt secrets from the CLI, using the `curl`, `echo`, `jq`, `gpg`, `grep` and `tr` CLI programs.
Here is an example:
```

```
curl https://app.renovatebot.com/renovate.pgp --output renovate.pgp
echo -n '{"o":"your-organization", "r":"your-repository (optional)", "v":"your-secret-value"}' | jq . -c | gpg --encrypt -a --recipient-file renovate.pgp | grep -v '^----' | tr -d '\n'
```

> The above script does this:
> 
> - Uses `curl` to download the Mend Renovate hosted app's public key
> - Echos a JSON object into `jq` for validation and compacting
> - Uses `gpg` to encrypt the contents
> 
> The `jq` step is optional, you could leave it out if you wish. Its primary value is validating that the string you input to `gpg` is valid JSON, and compact.

Reviewer: ```suggestion
The above script uses:

- `curl` to download the Mend Renovate hosted app's public key
- `echo` to echo a JSON object into `jq`
- `jq` to validate the JSON and then compact it
- `gpg` to encrypt the contents
- `grep` to ...
- `tr` to ...

The `jq` step is optional, you can leave it out if you wish.
Its primary value is validating that the string you echo to `gpg` is valid JSON, and compact.
```

Before you apply this suggestion, fill out the `...` parts to explain what that part of the command does. 😉

> Encrypted secrets typically have a single org, but you may encrypt a secret with more than one, e.g. specifying `org1,org2` to allow the secret to be used in both `org1` and `org2` organizations.

Reviewer: ```suggestion
Encrypted secrets usually have a single organization.
But you may encrypt a secret with more than one organization, for example: `org1,org2`.
This way the secret can be used in both the `org1` and `org2` organizations.
```

> For additional information about secrets use for private packages you can also read [Private package support](https://docs.renovatebot.com/getting-started/private-packages).

Reviewer: ```suggestion
For more information on how to use secrets for private packages, read [Private package support](https://docs.renovatebot.com/getting-started/private-packages).
```
